### PR TITLE
Add Tisha Soumya to the list

### PR DESCRIPTION
**Pull request by Tishasoumya-02**
Opening a PR to contribute to the repo.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/325"></a>

### CODE REVIEW

**Review by niteshseram — APPROVED** [review]
LGTM

File: public/data/Tishasoumya-02.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/Tishasoumya-02",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/SoumyaTisha",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (FayasNoushad) [inline public/data/Tishasoumya-02.json:17]: Add a line too after 17th line

Comment (FayasNoushad) [inline public/data/Tishasoumya-02.json:17]: Remove 2 whitespaces before `}`

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**